src/Identity/Core/src/CoreIdentityServiceCollectionExtensions.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Extensions.DependencyInjection
     /// <summary>
     /// Contains extension methods to <see cref="IServiceCollection"/> for configuring identity services.
     /// </summary>
-    public static class IdentityServiceCollectionExtensions
+    public static class CoreIdentityServiceCollectionExtensions
```

Comment:
Why rename? (This is a breaking change...)

Reply:
Looks okay otherwise.

Reply:
Never mind. Looked at the issue and realized it is an intentional break.

Reply:
Turns out this causes issues with templating tests which must not be rebuilding this, gonna back out this change for now